scripts/setup-vm-user.sh

```diff
@@ -6,18 +6,21 @@ LOGIN_PASS=$2
 # create the login user if it does not already exist
 if getent passwd $LOGIN_USER >/dev/null; then
   echo "Login User '$LOGIN_USER' already created..."
-else
-  # create user, set password and add it to the usual groups
-  adduser $LOGIN_USER --gecos "" --home "/home/$LOGIN_USER" --disabled-password
-  echo "$LOGIN_USER:$LOGIN_PASS" | chpasswd
-  usermod -a -G adm,cdrom,sudo,dip,plugdev,lpadmin,sambashare $LOGIN_USER
+  exit 0
+fi
 
-  # ensure the new user can do passwordless sudo
-  echo "$LOGIN_USER ALL=(ALL) NOPASSWD: ALL" | sudo tee /etc/sudoers.d/$LOGIN_USER
+# create user, set password and add it to the usual groups
+adduser $LOGIN_USER --gecos "" --home "/home/$LOGIN_USER" --disabled-password
+echo "$LOGIN_USER:$LOGIN_PASS" | chpasswd
+usermod -a -G adm,cdrom,sudo,dip,plugdev,lpadmin,sambashare $LOGIN_USER
 
-  # set the new user as the default in the login screen and lock the current (vagrant user's) screen session
-  if [[ $(which dm-tool) ]]; then
-    echo -e "[SeatDefaults]\nautologin-user=$LOGIN_USER" | sudo tee /etc/lightdm/lightdm.conf
-    XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0 dm-tool lock
```

Comment:
I remember that I locked the screen on purpose, but I don't remember why... hmmmm (*slaps himself for not writing a proper comment*)

Reply:
Based on the commit message from 2d5f3d143264047bf19f22b3118bc7fdb989cc00, I wanted to avoid that you accidentally use the VM with the vagrant user, where you actually should log out and log in as `$LOGIN_USER` instead. Locking the screen with the auto login user `$LOGIN_USER` somehow solved that.

Reply:
If the screen locking still works that way, I'd be happy to keep it

Reply:
this was done via `systemctl restart display-manager` now, right?

Reply:
yes, gnome-session-quit only works if called from the gnome session, if you try to call it in an ssh session you'll get an error, furthermore the current approach is independent from the used display manager and should work with any ubuntu version 15.04 or newer